File: rh-front/src/app/models/offreStage.ts
export class OffreStage {
    id : number;
    intitule : string;
    type_stage : string;
    duree_stage_mois : number;
    date_debut_stage : Date;
    remuneration : boolean;
    diplome_demande : string;
    descriptif_mission : string;

    constructor() {
        this.id = 0;
        this.intitule = '';
        this.type_stage = '';
        this.duree_stage_mois = 4;
        this.date_debut_stage = new Date();
        this.remuneration = false;
        this.diplome_demande = '';
        this.descriptif_mission = '';
    }
}

File: stagiaire/src/main/java/com/rh/stagiaire/Model/OffreStage.java
package com.rh.stagiaire.Model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

import javax.persistence.*;
import java.util.Date;

@Data @NoArgsConstructor @AllArgsConstructor @ToString
public class OffreStage {
    private Long id;
    private String intitule;
    private String type_stage;
    private Integer duree_stage_mois;
    private Date date_debut_stage;
    private Boolean remuneration;
    private String diplome_demande;
    private String descriptif_mission;

}